### PR TITLE
More ID work...

### PR DESCRIPTION
**Pull request by brmscheiner**
- integrates new annotation properties in match interface 
- hotfix for crashing individuals page 
- create individual page

### CODE REVIEW

File: src/pages/createIndividual/EncounterCard.jsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    data?.timeSpecificity,
+  );
+  const dataReady = !loading && !error;
+
+  return (
+    <Paper style={{ margin: '12px 0 20px', maxWidth: 660 }}>
+      <div style={{ display: 'flex', padding: 20 }}>
+        {data ? (
+          <img
+            src={defaultSightingSrc}
```
Comment (Atticus29) [inline src/pages/createIndividual/EncounterCard.jsx:54]: I assume that this is temporary?

File: src/pages/createIndividual/EncounterCard.jsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+              <Text
+                variant="body2"
+                id="SIGHTING_TIME_COLON"
+                values={{ date: sightingTime }}
+              />
+              <Text
+                variant="body2"
+                id="REPORTER_COLON"
+                values={{
+                  name: (
```
Comment (Atticus29) [inline src/pages/createIndividual/EncounterCard.jsx:98]: I'm glad we found this pattern lol.

File: src/pages/createIndividual/EncounterCard.jsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                    <Link href={`/users/${ownerGuid}`}>
```
Comment (Atticus29) [inline src/pages/createIndividual/EncounterCard.jsx:99]: newTab?

File: src/pages/match/MatchSighting.jsx [hunk 4]
```diff
@@ -229,6 +250,7 @@ export default function MatchSighting() {
           }}
           color="primary"
           variant="extended"
+          href={confirmMatchHref}
```
Comment (Atticus29) [inline src/pages/match/MatchSighting.jsx:253]: newTab?

File: src/pages/merge/IndividualCard.jsx [hunk 5]
```diff
@@ -53,10 +53,14 @@ export default function IndividualCard({ data, mergeConflicts }) {
           ) : (
             <Skeleton variant="text" height={30} width={140} />
           )}
-          {data && adoptionName ? (
-            <Text variant="body2">{adoptionName}</Text>
-          ) : (
-            <Skeleton variant="text" height={20} width={80} />
+          {adoptionName && (
```
Comment (Atticus29) [inline src/pages/merge/IndividualCard.jsx:56]: Why are adoptionName and data split into separate conditional expressions?

File: src/pages/match/MatchSighting.jsx [hunk 6]
```diff
@@ -109,6 +109,28 @@ export default function MatchSighting() {
...
+      const encounterGuid1 = selectedQueryAnnotation?.encounter_guid;
+      const encounterGuid2 = selectedMatchCandidate?.encounter_guid;
+      if (individualGuid1 && individualGuid2) {
+        return `/merge?i=${individualGuid1}&i=${individualGuid2}`;
+      } else if (individualGuid1 || individualGuid2) {
+        const individualGuid = individualGuid1 || individualGuid2;
+        const encounterGuid = individualGuid1
+          ? encounterGuid2
+          : encounterGuid1;
+        return `/assign-annotations?i=${individualGuid}&e${encounterGuid}`;
```
Comment (Atticus29) [inline src/pages/match/MatchSighting.jsx:126]: Missing = in e=

Comment (Atticus29) [reply src/pages/match/MatchSighting.jsx:126]: Also, individualGuid1 or individualGuid2?

**Review by Atticus29 — COMMENTED** [review]
Two breaking typos, and a few other little things.

File: src/pages/createIndividual/EncounterCard.jsx (hunk 2 above)
Comment (brmscheiner) [reply src/pages/createIndividual/EncounterCard.jsx:98]: absolutely!

File: src/pages/createIndividual/EncounterCard.jsx (hunk 3 above)
Comment (brmscheiner) [reply src/pages/createIndividual/EncounterCard.jsx:99]: sure

File: src/pages/match/MatchSighting.jsx (hunk 6 above)
Comment (brmscheiner) [reply src/pages/match/MatchSighting.jsx:126]: see line 122 for that

File: src/pages/match/MatchSighting.jsx (hunk 4 above)
Comment (brmscheiner) [reply src/pages/match/MatchSighting.jsx:253]: sure

File: src/pages/merge/IndividualCard.jsx (hunk 5 above)
Comment (brmscheiner) [reply src/pages/merge/IndividualCard.jsx:56]: the old way, if there was no adoptionName a skeleton would always show up. we only want the skeleton if there is an adoptionName that is loading

File: src/pages/createIndividual/EncounterCard.jsx [hunk 7]
```diff
@@ -0,0 +1,118 @@
+import React, { useMemo } from 'react';
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/pages/createIndividual/EncounterCard.jsx:1]: deleted the other file - it was unused (leftover from fake data days)

**Review by Atticus29 — APPROVED** [review]
LGTM! Seems to be working as expected as well!

**Pull request merged**